packages/ui/cart/drawer/CartDrawer.tsx

```diff
@@ -39,12 +39,23 @@ import { useCartDrawer } from './hooks/useCartDrawer';
 
 export function CartDrawer() {
    const appContext = useAppContext();
-   const { isOpen, onOpen, onClose, onViewCart } = useCartDrawer();
+   const {
+      isOpen,
+      onOpen,
+      onClose,
+      onViewCart,
+      errorMessages,
+      clearErrorMessages,
+   } = useCartDrawer();
    const isMounted = useIsMountedState();
    const cart = useCart();
    const checkout = useCheckout();
    const isCartEmpty = cart.isFetched && !cart.data?.hasItemsInCart;
-
+   const clearAddToCartErrors = () => clearErrorMessages();
```

Comment:
Can we use `clearErrorMessages` directly without wrapping it? I think `clearAddToCartErrors` might be misleading as a name, since this would not just clear "add to cart" errors, but really any error displayed in the drawer